### ISSUE TITLE
Standalone BIOS Update does not complete

### ISSUE THREAD

**Issue by dsanders11** [FrameworkComputer/SoftwareFirmwareIssueTracker#20]
### System Model or SKU

Please select one of the following

- [ ] Framework Laptop 13 (11th Gen Intel® Core™)
- [ ] Framework Laptop 13 (12th Gen Intel® Core™)
- [ ] Framework Laptop 13 (13th Gen Intel® Core™)
- [ ] Framework Laptop 13 (AMD Ryzen™ 7040 Series)
- [x] Framework Laptop 13 (Intel® Core™ Ultra Series 1)
- [ ] Framework Laptop 16 (AMD Ryzen™ 7040 Series)

### BIOS VERSION

3.01 (updating to 3.03)

### DIY Edition information
If you are experiencing an issue on a DIY system, Please also fill out the memory and storage devices you are using. 

Memory: [Crucial 32GB DDR5-5600 SODIMM (CT32G56C46S5)](https://www.crucial.com/memory/ddr5/ct32g56c46s5)
Storage: [WD_BLACK SN770 SSD - 2TB (WDS200T3X0E-00B3N0)](https://shop.sandisk.com/products/ssd/internal-ssd/wd-black-sn770-nvme-ssd?sku=WDS200T3X0E-00B3N0)

### Port/Peripheral information

1. USB-C
2. USB-C
3. HDMI
4. USB-C

### Standalone Operation

Are you running your mainboard as a standalone device. Is standalone mode enabled in the BIOS?
- [x] Yes
- [ ] No

## Describe the bug

When updating the BIOS to 3.03 (on the way to 3.04) ([following the instructions here](https://knowledgebase.frame.work/en_us/framework-laptop-bios-and-driver-releases-intel-core-ultra-series-1-H1nZQdxYR)), the system is currently hanging indefinitely after switching the USB-C power to the opposite side. It hangs on the "Please keep AC attached while updating" screen and also displays "PD1: 0_0_8" on the screen.

The BIOS update was started with the EFI shell on a USB drive. I originally attempted to install from Windows 11 but that always hung on the "Please keep AC attached while updating" screen as well (with a few more version numbers displayed), and would never proceed. Using the EFI shell I was able to start the update process, but now it is hanging just as when attempting to run from Windows 11.

Switching the power back to the original side displays the message to power down and switch sides. So the board is not completely bricked, it's just stuck in the intermediate state and can't finish the update as a result.

I am unable to boot into the BIOS or do anything other than see those two screens, depending on which side I plug the power into.

## Steps To Reproduce

1. Run board in standalone mode with power plugged into left side
2. Begin BIOS update to 3.03/3.04 using the EFI shell on a USB drive
3. Wait for update to complete and prompt you to switch side power is plugged into
4. Power down board and plug power into opposite side
5. Observe "Please keep AC attached while updating" screen on boot, which hangs indefinitely

## Expected behavior

BIOS update should not hang indefinitely on "Please keep AC attached while updating" screen.

## Screenshots

N/A

## Operating System (please complete the following information):

 - OS/Distribution: Windows 11

**Comment by dsanders11** [comment]
I've also tried pulling the SSD in case update files placed in the EFI partition were interfering, but no behavior changes between running it with or without the SSD.

**Comment by dsanders11** [comment]
Also further information that I forgot to include in the original description, after the initial part of the BIOS update proceeded, the board started doing the flashing red and blue lights like it did before standalone mode was enabled when I first got it. I have a suspicion that during the BIOS update process it may have lost state that it was in standalone mode? Perhaps that's what's blocking the update from continuing.

**Comment by dsanders11** [comment]
I was able to resolve this issue and continue the BIOS update by changing monitors, perplexingly enough, after a few back and forths with support. I'm using the Framework HDMI (3rd generation) expansion card, and I was originally using a Dell U4320Q (4K) which was displaying output, but not continuing the update. I changed to an old 1080p monitor I happened to have lying around and immediately on boot the progress bar appeared where it had previously not and the BIOS update proceeded.

So there appears to be some issue regarding the monitor connected to the HDMI expansion card which can cause BIOS updates to not proceed. Between the two monitors the immediately obvious differences are native resolution (4K vs 1080p) and HDMI spec version (2.0 vs 1.4).

If anyone else comes across this issue with the same issue, try changing to a different monitor (and make a comment with the specs if it resolves the issue). Closing as this has been resolved.
